content/docs/studio/get-started.md

```diff
@@ -1,3 +1,12 @@
+<admon>
+
+**We have renamed Views to Projects in Iterative Studio.**
+
+Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
+settings_ are now called _Project settings_; and so on.
+
+</admon>
+
```

Comment:
I don't see a need to warn someone who will read the Get Started (new user) about changes to previous versions. Kinda scary that it's so prominent too 🙂 

```suggestion
```

Reply:
Sounds good, thanks.